src/main/java/co/org/osso/api/BookService.java

```diff
@@ -0,0 +1,52 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class BookService {
+
+    Map<String, Book> mapBooks = new HashMap<>();
+    int id;
+
+    public Book createBook(Book book) {
+        book.setBookID(""+id++);
+        mapBooks.put(book.getBookID(), book);
+        return book;
+    }
+
+    public Book getBook(String id) {
+        if (mapBooks.get(id) != null){
```

Comment:
There is a shorthand notation for doing all these 5 lines.
```suggestion
        return mapBooks.getOrDefault(id, new Book());
```

Reply:
Good tip, thank you!